sale_order_volume/models/sale_order.py

```diff
@@ -23,7 +23,26 @@ class ProductCategoryVolume(models.Model):
     volume = fields.Float(
         string='Volume (m³)',
     )
+    pallet_nb = fields.Integer(
+        string="Number of pallets",
+        compute='compute_pallet_nb',
+        store=True
+    )
+
+    @api.model
+    def get_pallet_volume_indication(self):
+        pallet_volume = float(self.env["ir.config_parameter"].get_param(
+        'sale_order_volume.pallet_volume'
+        )) or 0
+        return "One pallet = %.2f m³" % pallet_volume
 
+    @api.multi
+    @api.depends('volume')
+    def compute_pallet_nb(self):
+        pallet_volume = float(self.env["ir.config_parameter"].get_param(
+            'sale_order_volume.pallet_volume'
+        )) or 0
+        self.pallet_nb = self.volume // pallet_volume
```

Comment:
If pallet_volume is None, you'll get a `ZeroDivisionError`

Reply:
Indeed ! Changed for
```python
        if pallet_volume:
            self.pallet_nb = self.volume // pallet_volume
        else:
            self.pallet_nb = 0
```